CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+## [0.11.1] - 2022-07-19
+### Added
+- config/server/ - add empty init file in config/server.
```

Comment:
No need to mention this, definitely not in "added", it's not an added feature...
If this is made to fix some issue, then we can mention it under "fixed"
@SatyaBade12 ?

Reply:
Yes from what I understood it is to make the Entropy docs work, I'll put it in fixed then